lib/game.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+# Game class
+class Game
+  include UI
+
+  attr_accessor :continue
+  def initialize(board, player1, player2)
+    @new_board = board
+    @player1 = player1
+    @player2 = player2
+  end
+
+  def play
+    until game_is_over
+      @player1.make_move(@new_board) unless has_won?(@player2) || !@new_board.not_full?
```

Comment:
Great job here! We can now further trim this into:
```ruby
@player1.make_move(@new_board) unless game_is_over
```
You will observe that `game_is_over` has the exact same logic you attached in your `unless` statement :)